website/javascript/templates/GameFeed.vue

```diff
@@ -114,8 +114,10 @@ export default {
                 const players = Object.values(game.players).sort((r1, r2) => {
                   return r1.rank - r2.rank
                 })
-
                 game.playerSorted = players
+
+                // include the initial 3 ships of each player
+                game.total_ships = game.ships_produced + (players.length * 3);
```

Comment:
Why is it hard coded ? 3 is not in a global variable or something ?

Reply:
Mostly since so far as I know the game constants aren't available at this point, it's unlikely to be needed anywhere else besides this line and is very unlikely to change. But if someone wants to pull it out into a constant feel free.